app/transforms/node-license.ts

```diff
@@ -17,10 +22,7 @@ export default class NodeLicenseTransform extends DS.Transform {
         const {
             copyrightHolders = [],
             year = '',
-        } = camelizeKeys(value) as {
-            copyrightHolders?: string | string[],
-            year?: string,
-        };
+        } = camelizeKeys(value as Record<'copyright_holders|year', unknown>) as DeserializedNodeLicense;
```

Comment:
should be `Record<'copyright_holders' | 'year': unknown }` -- but really, `value` should be fine as as a `SerializedNodeLicense`...
```suggestion
        } = camelizeKeys(value) as DeserializedNodeLicense;
```
but i think `camelizeKeys` has the wrong types... and i got nerd-sniped -- i think this would do it:
```
export function camelizeKeys<K extends string>(obj: Partial<Record<K, unknown>>) {
```

Reply:
I thought value was fine as typed in the `deserialize` signature but w/o the type casting. I get the following type error
```
Argument of type 'SerializedNodeLicense' is not assignable to parameter of type 'Record<string, unknown>'.
  Index signature is missing in type 'SerializedNodeLicense'
```